.vscode/launch.json

```diff
@@ -13,7 +13,7 @@
       "args": [
         "--opts",
         "${workspaceRoot}/test/mocha.opts",
-        "packages/*/dist/test/**/*.js",
+        "packages/*/dist*/test/**/*.js",
```

Comment:
If there are both `dist8` and `dist10`, the tests will be run twice.

Reply:
@virkt25 that's a good point, `build` package replaces `DIST` with the real `dist*` folder when run tests, which is the most accurate match, but the same approach is not achievable in a json config file. Trigger `npm test` is a good solution. While....

> Would it be a better idea to just trigger the npm test under the hood

Did a quick search, there seems no easy way to run a npm script in the vscode launch.
And BTW, the build doesn't fail actually, the failure I observed should be a random one. And appveyor does clean the cache per run.

Reply:
This issue is kind of brought up in #1348 as well. The gist is that I tried running lb-mocha script instead of _mocha and had `DIST` in place of `dist*`. This works fine until you try to hit the restart button while in the middle of debugging. What happens is that one of the child (the mocha tests) or the parent (the lb-mocha script) processes do not shutdown properly and the whole thing times out.

Reply:
I didn't mean a build failure but rather the `@loopback/build` package tests aren't covered in the current wildstar since like the `@loopback/cli` it's a JS package and doesn't have a `dist` folder.

Reply:
I see, I made this quick PR just to get the debug run for `@loopback/rest`, and I believe it also works for any ts package that generates a `dist*` when built. I didn't realize it doesn't even run tests for js package.

How about we work on a perfect fix in issue https://github.com/strongloop/loopback-next/issues/1348, and merge this PR as workaround, running tests for all ts packages is better than none lol.